slides/ds101-worskhop.mdx

```diff
@@ -137,6 +141,12 @@ Shared vocabulary is very important, it results in more productive cooperative w
 
 ---
 
+# Review
+
+[Mockups. The full version](https://www.figma.com/file/CCIfruTLVYWanYldiUvvYv/Naming-Ex?node-id=1%3A2)
```

Comment:
@okonet iframe is not allowed, so we cannot embed.

Reply:
Solved in https://github.com/component-driven/ds-101-workshop/pull/24